Makefile

```diff
@@ -98,6 +102,9 @@ func-test:
 check-env:
 	@./scripts/check-env.sh
 
+commitlint:
+	commitlint --from $(GIT_SINCE)
```

Comment:
@nixpanic can you run this test in CI? is it already running?

Reply:
it is not part of the CI yet, this is only added for contributors to check locally

It can get added to Travis CI, but for that I prefer to have this merged first.